python/py-thefuck/Portfile

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+set real_name       thefuck
+name                py-${real_name}
```

Comment:
Please change the port so that its name doesn't have the "py-" prefix as Herby mentioned above and as I mentioned in the ticket that requested this port.

PRs aren't a good place to have a discussion about proposing changes in how portgroups work. The way the python portgroup works now is how we want it to work, but if you'd like to make a case for changing that, please do so on the macports-dev mailing list so that everyone can participate.

Reply:
py- removed. √